### ISSUE TITLE
`rpki-signed-objects.xml` doesn't have `<title>` element

### ISSUE THREAD

**Issue by ronaldtse** [ietf-ribose/bibxml-data-iana#3]
This file doesn't have `<title>` element: https://github.com/ietf-ribose/iana-registries/blob/main/rpki-signed-objects/rpki-signed-objects.xml

_Originally posted by @andrew2net in https://github.com/ietf-ribose/bibxml-data-iana/issues/2#issuecomment-979207262_

**Comment by ronaldtse** [comment]
The "RPKI Signed Objects" description is now provided under `rpki.xml`:
https://github.com/ietf-ribose/iana-registries/blob/b471f785838bc5ef495134b005e153189d66fdc3/rpki/rpki.xml#L9-L34

HTML page:
https://www.iana.org/assignments/rpki/rpki.xhtml#signed-objects

This makes me wonder if we should split off the registries. I believe every register should be independently citable.

**Comment by ronaldtse** [comment]
In the current bibxml8 implementation it just gives an empty `<title/>` element for this registry because it is moved inside `rpki.xml`.

https://xml2rfc.tools.ietf.org/public/rfc/bibxml8/reference.IANA.rpki-signed-objects.xml

```xml
<reference anchor="RPKI-SIGNED-OBJECTS" target="http://www.iana.org/assignments/rpki-signed-objects">
<front>
<title/>
<author>
<organization>IANA</organization>
</author>
<date/>
</front>
</reference>
```

I've created a generic concern here which will solve this issue properly: https://github.com/relaton/relaton-iana/issues/2

**Comment by andrew2net** [comment]
fixed in relaton/relaton-iana#2
